### ISSUE TITLE
upgradeToRevocableSession support

### ISSUE THREAD

**Issue by montaro** [ParsePlatform/parse-server#1056]
I can't do the POST request that upgrades the legacy token to the revocable token, is it supported in the open source version?

```
curl -X POST \
  -H "X-Parse-Application-Id: <YOUR_APPLICATION_ID>" \
  -H "X-Parse-REST-API-Key: <YOUR_REST_API_KEY>" \
  -H "X-Parse-Session-Token: <CURRENT_LEGACY_SESSION_TOKEN>" \
  http://localhost:1337/parse/upgradeToRevocableSession
```

**Comment by flovilmart** [comment]
legacy sessions are not supported on parse-server. Therefore, only revocable sessions are supported. This endpoint is not implemented.

**Comment by montaro** [comment]
Thanks!

**Comment by ronnno** [comment]
I understand that upgradeToRevocableSession is not implemented and that is fine. but our mobile client keeps calling this url against my server. I think we followed the migration guide on this subject. but how can we get the clients to stop sending this request?

**Comment by drewlarsen** [comment]
@ronnno The fix is to remove `ParseUser.enableRevocableSessionInBackground();` from your Android client app.

As a temporary fix, we were also able to address this by adding middleware in our ParseServer setup:
```        var path = req.path.split('/');
        if (path[1] === 'parse') {
            // respond to spurious /parse requests
            if (path.length > 2) {
                if (path[2] === 'upgradeToRevocableSession') {
                    res.status(200).end();
                }
            }
            return;
        }```

**Comment by flovilmart** [comment]
it's been implemented in 2.2.19 ! Upgrade your parse-server's and it should work
